File: app/src/main/java/ai/kortnevdmitriy/msafiri/activities/Account.java
package ai.kortnevdmitriy.msafiri.activities;

import android.net.Uri;
import android.os.Bundle;
import android.widget.TextView;

import androidx.appcompat.app.AppCompatActivity;
import androidx.appcompat.widget.Toolbar;

import com.google.firebase.auth.FirebaseAuth;
import com.google.firebase.auth.FirebaseUser;

import ai.kortnevdmitriy.msafiri.R;
import de.hdodenhof.circleimageview.CircleImageView;

public class Account extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_account);
        Toolbar toolbar = findViewById(R.id.toolbar);
        setSupportActionBar(toolbar);
        getSupportActionBar().setDisplayHomeAsUpEnabled(true);

        FirebaseUser user = FirebaseAuth.getInstance().getCurrentUser();
        String displayName = user.getDisplayName();
        String userEmail = user.getEmail();
        String phoneNumber = user.getPhoneNumber();
        Uri photoUrl = user.getPhotoUrl();


        TextView tvDisplayName = findViewById(R.id.user_profile_name);
        TextView tvDisplayBio = findViewById(R.id.user_profile_short_bio);
        TextView tvDisplayEmail = findViewById(R.id.user_profile_email);
        TextView tvDisplayPhone = findViewById(R.id.user_profile_phone);
        CircleImageView avatarImageView = findViewById(R.id.user_profile_photo);

    /*    Picasso.with(getApplicationContext())
                .load(photoUrl)
                .into(avatarImageView);*/
        tvDisplayName.setText(displayName);
        tvDisplayEmail.setText(userEmail);
        tvDisplayPhone.setText("+254722972233");
    }

}

File: app/src/main/java/ai/kortnevdmitriy/msafiri/activities/Account.kt
package ai.kortnevdmitriy.msafiri.activities

import ai.kortnevdmitriy.msafiri.R
import android.os.Bundle
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity
import androidx.appcompat.widget.Toolbar
import com.google.firebase.auth.FirebaseAuth
import com.squareup.picasso.Picasso
import de.hdodenhof.circleimageview.CircleImageView

class Account : AppCompatActivity() {
	
	override fun onCreate(savedInstanceState: Bundle?) {
		super.onCreate(savedInstanceState)
		setContentView(R.layout.activity_account)
		val toolbar = findViewById<Toolbar>(R.id.toolbar)
		setSupportActionBar(toolbar)
		supportActionBar?.setDisplayHomeAsUpEnabled(true)
		
		val user = FirebaseAuth.getInstance().currentUser
		val displayName = user?.displayName
		val userEmail = user?.email
		val phoneNumber = user?.phoneNumber
		val photoUrl = user?.photoUrl
		
		
		val tvDisplayName = findViewById<TextView>(R.id.user_profile_name)
		val tvDisplayBio = findViewById<TextView>(R.id.user_profile_short_bio)
		val tvDisplayEmail = findViewById<TextView>(R.id.user_profile_email)
		val tvDisplayPhone = findViewById<TextView>(R.id.user_profile_phone)
		val avatarImageView = findViewById<CircleImageView>(R.id.user_profile_photo)
		Picasso.get().load(photoUrl).into(avatarImageView)
		tvDisplayName.text = displayName
		tvDisplayEmail.text = userEmail
		tvDisplayPhone.text = phoneNumber
	}
	
}
